schema/model.json

```diff
@@ -23,9 +23,13 @@
             "format": {
               "name": "format",
               "type": "string",
-              "description": "Schema format identifier for this schema version",
-              "clientrequired": true,
-              "serverrequired": true
+              "description": "Schema format identifier for this schema version"
+            },
+            "validation": {
+              "name": "validation",
+              "type": "boolean",
+              "description": "Verify compliance with specified 'format'",
```

Comment:
Should this be:

- 'format'
- envelope format (no quotes)
- 'envelope'

? There are other occurrences of the same thing - I haven't commented on them.

Reply:
I changed it to "schema `format`".